analyzers/python/integration_test.go

```diff
@@ -0,0 +1,133 @@
+package python_test
+
+import (
+	"flag"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/apex/log"
+
+	"github.com/fossas/fossa-cli/analyzers"
+	"github.com/fossas/fossa-cli/exec"
+	"github.com/fossas/fossa-cli/module"
+	"github.com/fossas/fossa-cli/pkg"
+	"github.com/fossas/fossa-cli/testing/fixtures"
+	"github.com/stretchr/testify/assert"
+)
+
+var pythonAnalyzerFixtureDir = filepath.Join(fixtures.Directory(), "python", "analyzer")
+
+func TestMain(m *testing.M) {
+	// flags are not parsed at this point. In order to have testing.Short() read actually provided values, this must be executed
+	flag.Parse()
+	if testing.Short() {
+		return
+	}
+
+	err := fixtures.Clone(pythonAnalyzerFixtureDir, projects)
+	if err != nil {
+		panic(err)
+	}
+
+	sentryLocation := filepath.Join(pythonAnalyzerFixtureDir, "sentry")
+	stdout, stdErr, err := exec.Run(exec.Cmd{
+		Name:    "pip",
+		Argv:    []string{"install", "-r", "requirements-base.txt"},
+		Dir:     sentryLocation,
+		Command: "pip",
+	})
+	if err != nil {
+		log.Error(stdout)
+		log.Error(stdErr)
+		panic(err)
+	}
+	_, stdErr, err = exec.Run(exec.Cmd{
+		Name:    "pip",
+		Argv:    []string{"install", "-r", "requirements-dev.txt"},
+		Dir:     sentryLocation,
+		Command: "pip",
+	})
+	if err != nil {
+		log.Error(stdErr)
+		panic(err)
+	}
+
+	exitCode := m.Run()
+	defer os.Exit(exitCode)
+}
+
+func TestPythonAnalysis(t *testing.T) {
+	t.Parallel()
+	for _, project := range projects {
+		proj := project
+		projDir := filepath.Join(pythonAnalyzerFixtureDir, proj.Name)
+		t.Run("Analysis:"+proj.Name, func(t *testing.T) {
+			t.Parallel()
+			module := module.Module{
+				Dir:         projDir,
+				Type:        pkg.Python,
+				Name:        proj.Name,
+				Options:     map[string]interface{}{"strategy": "deptree"},
+				BuildTarget: projDir,
+			}
+
+			if proj.Name == "sentry" {
+				module.Options["strategy"] = "deptree"
+			}
+
+			analyzer, err := analyzers.New(module)
+			assert.NoError(t, err)
+
+			deps, err := analyzer.Analyze()
+			assert.NoError(t, err)
+
+			assert.NotEmpty(t, deps.Direct)
+			assert.NotEmpty(t, deps.Transitive)
+		})
+	}
+}
+
+var projects = []fixtures.Project{
+	fixtures.Project{
+		Name:   "vibora",
+		URL:    "https://github.com/vibora-io/vibora",
+		Commit: "51d6691a430bbb6ac57df54dc7d8276f62953bb2",
+	},
+	// ansible is failing checkout to a commit with the error: worktree contains unstaged changes, but I cannot figure out why
```

Comment:
That's bizarre. I'm alright with this merging anyway, but I'm also down to pair on this.